xdebug_gc_stats.c

```diff
@@ -220,3 +230,16 @@ PHP_FUNCTION(xdebug_start_gcstats)
 	}
 }
 /* }}} */
+
+/* {{{ proto void xdebug_stop_gcstats()
+   Stop collecting garbage collection statistics */
+PHP_FUNCTION(xdebug_stop_gcstats)
+{
+	if (XG(gc_stats_enabled) == 1) {
+		RETVAL_STRING(XG(gc_stats_filename));
+		xdebug_gc_stats_stop();
+	} else {
+		RETVAL_FALSE;
+		php_error(E_NOTICE, "Function trace was not started");
```

Comment:
"Function trace"? You mean "Garbage Collection Statistics" :-) (Check whether you phrase the term the same way as when "GCS was already started".